Description: enable the otvalid and gxvalid table validation modules
 The 'ftvalid' demo in the freetype2-demos package is only useful if certain
 extra extensions are enabled in freetype.  These extensions are not enabled
 upstream by default, but there's no reason for us not to enable them.
Author: Paul Wise <pabs@debian.org>
Bug-Debian: http://bugs.debian.org/520879
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/freetype/+bug/239626
Forwarded: not-needed

Index: freetype-2.7.1/modules.cfg
===================================================================
--- freetype-2.7.1.orig/modules.cfg	2017-03-08 18:36:54.788962859 +0100
+++ freetype-2.7.1/modules.cfg	2017-03-08 18:37:21.477090060 +0100
@@ -120,7 +120,7 @@
 # TrueType GX/AAT table validation.  Needs ftgxval.c below.
 #
 # No FT_CONFIG_OPTION_PIC support.
-# AUX_MODULES += gxvalid
+AUX_MODULES += gxvalid
 
 # Support for streams compressed with gzip (files with suffix .gz).
 #
@@ -143,7 +143,7 @@
 # OpenType table validation.  Needs ftotval.c below.
 #
 # No FT_CONFIG_OPTION_PIC support.
-# AUX_MODULES += otvalid
+AUX_MODULES += otvalid
 
 # Auxiliary PostScript driver component to share common code.
 #
